.github/workflows/test.yml

```
@@ -83,6 +83,8 @@ jobs:
           SPRING_JPA_DATABASE_PLATFORM: org.hibernate.dialect.PostgreSQLDialect
           SPRING_TEST_DATABASE_REPLACE: NONE
           SPRING_DATA_REDIS_HOST: localhost
+          env:
+            SPRING_TASK_SCHEDULING_ENABLED: false
```

actionlint (1.7.12)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)